File: python3/168.py
class Solution:
    def convertToTitle(self, c: int) -> str:
        x = "ZABCDEFGHIJKLMNOPQRSTUVWXY"
        s = []
        while c > 0:
            s.append(x[c % 26])
            c = int((c - 1) / 26)
        return "".join(s[::-1])

File: javascript/168.js
/**
 * @param {number} c
 * @return {string}
 */
var convertToTitle = function (c) {
    let s = [];
    let x = "ZABCDEFGHIJKLMNOPQRSTUVWXY";
    while (c > 0) {
        s.push(x[c % 26]);
        c = parseInt((c - 1) / 26);
    }
    return s.reverse().join("");
};
